bin/cheritest/cheritest_vm.c

```diff
@@ -312,6 +313,19 @@ xfail_need_writable_tmp(const char *name __unused)
 	return (reason);
 }
 
+const char* xfail_need_writable_non_tmpfs_tmp(const char *name)
+{
+	struct statfs info;
```

Comment:
Style(9) would like a blank line between local variable definitions and the start of code.